forgerock-openbanking-uk-aspsp-common/src/main/java/com/forgerock/openbanking/common/model/openbanking/persistence/account/FRAccountAccessConsent.java

```diff
@@ -52,6 +53,7 @@
     @Indexed
     public String id;
     private FRReadConsentResponse accountAccessConsent;
+    private FRCustomerInfo customerInfo;
     @Indexed
```

Comment:
Added Customer info as additional data in the account access Consent for customer info consent.